app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+file_name = input("Please enter the name of the file: ")
+file_content = ""
+
+while True:
+    new_line = input("Waiting for input: ")
+    if new_line == "stop":
+        break
+    else:
+        file_content += new_line + "\n"
+
+with open(file_name + ".txt", "w") as file:
+    file.writelines(file_content)
```

Comment:
```suggestion
    file.write(file_content)
```
file_content has an str type. It's better to use the `write` method with strings. Use `writelines` with iterables like a list
Exmaple:
```
file.writeline(["hello", " ", "world", "!"])
```

Reply:
Fixed